### ISSUE TITLE
Not able to login getting Connection to Play Services Failed

### ISSUE THREAD

**Issue by srinivasvarma460** [youtube/yt-watchme#49]
Connection to Play Services Failed, error: 4, reason: ConnectionResult{statusCode=SIGN_IN_REQUIRED, resolution=PendingIntent{411df8f: android.os.BinderProxy@20fc316}, message=null}

**Comment by LaoMengFlutter** [comment]
Hello,I have the same problem，How did you solve it?

**Comment by ismaelCespedes** [comment]
+1

**Comment by vranacse** [comment]
first client.json file is req. to put in app folder ! ,   you will get it when you enable the google plus api and youtube data api from google api console 
@srinivasvarma460  @781238222 @ismaelCespedes

**Comment by MuthuEphron** [comment]
Hi guys, 
Pleas help this issues #53

**Comment by TomACPace** [comment]
@vranacse No, that's incorrect.  Following your advice, same error.  Also, client.json is not talked about in README.md  so that should not be a requirement.  I'm investigating this further.

**Comment by TomACPace** [comment]
I believe a critical missing component to this yt-watchme and yt-direct-lite-android projects is the unspecified requirement for debug builds to be signed.  Unsigned debug builds will not work.  I spent a lot of time trying all manner of configurations.   Final testing: a signed debug build with an API key for restricted access worked, then reverting from using a restricted API key, but still with signed debug build also worked.

**Comment by cyberformed** [comment]
Guys, I have tried all the configurations and all methods that mentioned, but it seems to be nothing worked at all.

As I have seen that, the youtube developers team is also not providing the right answers/ support to it.
